README.md

```diff
@@ -238,29 +238,6 @@ The client keys used with Parse are no longer necessary with Parse Server. If yo
 
 #### Advanced options
```

Comment:
remove the 

`### Advanced options `

Reply:
`##### Logging` is still one layer down from `#### Advanced options`

Reply:
Logger is there twice

Reply:
I'll do a separate PR to clean up the ReadMe, there are a few dead links and structure can be clearer.